scripts/genbindings.sh

```diff
@@ -28,6 +28,24 @@ function checkInstalledLocally() {
   fi
 }
 
+function compileAndMoveTo() {
```

Comment:
That the files are moved are an implementation detail of this function IMO, so I'd just call it 
```suggestion
function compileTo() {
```
;) Also, right now they're copied.